src/main/java/UI/Model/ProductDto.java

```diff
@@ -0,0 +1,53 @@
+package UI.Model;
+
+import enums.Category;
+
+public class ProductDto {
+
+    private String _productName;
+    private Category _category;
+    private double _price;
+    private int _productQuantity;
+
+    // Constructor
+    public ProductDto(String productName, Category category, double price, int productQuantity) {
```

Comment:
nice

Reply:
think about if needed to add discounts or conditions about the product we want to view in UI